src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/ProvidePublicParameterlessSafeHandleConstructor.Fixer.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Analyzer.Utilities;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Editing;
+
+namespace Microsoft.NetCore.Analyzers.InteropServices
+{
+    public abstract class ProvidePublicParameterlessSafeHandleConstructorFixer : CodeFixProvider
```

Comment:
Could be made sealed instead of abstract and add `[ExportCodeFixProvider(LanguageNames.CSharp, LanguageNames.VisualBasic), Shared]`, and delete derived classes.

Reply:
I did not know we could do that. I'll switch it.